ogr/ogrsf_frmts/pg/ogrpgdatasource.cpp

```diff
@@ -2355,6 +2376,62 @@ OGRLayer *OGRPGDataSource::GetLayer(int iLayer)
         return papoLayers[iLayer];
 }
 
+/************************************************************************/
+/*                             FindSchema()                             */
+/************************************************************************/
+
+// Check that the schema exist. If there is a single match in a case
+// insensitive way, use it. Otherwise error out if th match is not exact.
+// Return the schema name with its exact case from pg_catalog, or an empty
+// string if an error occurs.
+std::string OGRPGDataSource::FindSchema(const char *pszSchemaNameIn)
```

Comment:
consider returning `std::optional<std::string>` for more clarity at the call site?

Reply:
done